notebooks/106-speech-recognition-quantize-wav2vec2/106-speech-recognition-quantize-wav2vec2.ipynb

```diff
@@ -0,0 +1,631 @@
+    "torch_model = Wav2Vec2ForCTC.from_pretrained(Path(MODEL_DIR)/'pytorch')\n",
+    "onnx_model_path = Path(MODEL_DIR)/\"wav2vec2_base.onnx\"\n",
+    "if not onnx_model_path.exists():\n",
+    "    export_model_to_onnx(torch_model, onnx_model_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d6911365",
+   "metadata": {
+    "id": "sNWDAGGd0uRt"
+   },
+   "source": [
+    "## Convert the ONNX Model to OpenVINO IR"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20aeea80",
+   "metadata": {
+    "id": "-6P0c_960uR5",
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "ir_model_xml = onnx_model_path.with_suffix(\".xml\")\n",
+    "ir_model_bin = onnx_model_path.with_suffix(\".bin\")\n",
+    "\n",
+    "if not ir_model_xml.exists():\n",
+    "    !mo --input_model $onnx_model_path --output_dir $MODEL_DIR --model_name wav2vec2_base --input inputs --input_shape [1,-1] --data_type FP16"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "635f4b0d",
+   "metadata": {
+    "id": "LBbY7c4NsHzT"
+   },
+   "source": [
+    "## Prepare LibriSpeech Dataset\n",
+    "\n",
+    "The code below will download and unpack the archive with 'test-clean' subset of LibriSpeech Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "43070514",
+   "metadata": {
+    "id": "NN-qRME1a-Sm"
+   },
+   "outputs": [],
+   "source": [
+    "download_file(\"http://openslr.elda.org/resources/12/test-clean.tar.gz\", directory=DATA_DIR, show_progress=True)\n",
+    "\n",
+    "if not os.path.exists(f'{DATA_DIR}/LibriSpeech'):\n",
+    "    with tarfile.open(f\"{DATA_DIR}/test-clean.tar.gz\") as tar:\n",
+    "        tar.extractall(path=DATA_DIR)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da802acd",
+   "metadata": {
+    "id": "E5hsOsj-0uSc"
+   },
+   "source": [
+    "## Define DataLoader for POT\n",
+    "In this step, we need to define `DataLoader` based on POT API. It will be used to collect statistics for quantization and run model evaluation.\n",
+    "Wav2Vec2 model accepts a raw waveform of the speech signal as input and produces vocabulary class estimations as output.\n",
+    "\n",
+    "Consider increase `samples_limit` to get more precise results, suggested value is `300` or more, it will take longer time to process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2a378cb",
+   "metadata": {
+    "id": "6xnl2PhM0uSn",
+    "tags": [],
+    "test_replace": {
+     "return len(self._ds)": "return 3"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "class LibriSpeechDataLoader(POTDataLoader):\n",
+    "    samples_limit = 50\n",
+    "    # Required methods\n",
+    "    def __init__(self, config):\n",
+    "        \"\"\"Constructor\n",
+    "        :param config: data loader specific config\n",
+    "        \"\"\"\n",
+    "        if not isinstance(config, Dict):\n",
```

Comment:
@KodiaqQ, demo-notebooks #105 and #113 contain this useless type checking, I only left it for consistency. Since you noticed it, I'm considering to remove this check.